src/porepy/numerics/elliptic.py

```diff
@@ -141,6 +190,66 @@ def save(self, variables, save_every=None):
             variables = {k: self._data[k] for k in variables if k in self._data}
         export_vtk(self.grid(), file_name, variables, folder=folder_name)
 
+
+    ### Helper functions for linear solve below
+    def _setup_preconditioner(self):
```

Comment:
I assume you have some plan for the functions below, but you do realize this will never be called as you have commented it out?